### PR TITLE
Remove parenthesis next to commas

### PR DESCRIPTION
**Pull request by othercorey**
Fixes https://github.com/cakephp/cakephp-codesniffer/issues/351

### CODE REVIEW

File: CakePHP/Tests/Commenting/TypeHintUnitTest.1.inc [hunk 1]
```diff
@@ -36,4 +36,7 @@ function test()
 
     /** @psalm-var null|scalar|class-string<\Cake\I18n\Number>|array $testVar2 */
     $testVar2;
+
+    /** @var string|null|array<string|int, array<mixed, array<string>|string|null>|string|null>
```
Comment (dereuromark) [inline CakePHP/Tests/Commenting/TypeHintUnitTest.1.inc:40]: This seems to be missing closing tag though

Comment (othercorey) [reply CakePHP/Tests/Commenting/TypeHintUnitTest.1.inc:40]: It's impossible to read, but after breaking it down manually, it should be right.

Comment (dereuromark) [reply CakePHP/Tests/Commenting/TypeHintUnitTest.1.inc:40]: I mean the closing tag of */
For the comment to be valid docblock

Comment (othercorey) [reply CakePHP/Tests/Commenting/TypeHintUnitTest.1.inc:40]: oh yea! good catch!

**Pull request merged**